workflow/tests/ACCA_Examples/Bob_Ross_Residence.xml

```diff
@@ -0,0 +1,1483 @@
+            <InteriorShading>
+              <SystemIdentifier id='WindowBedroom3WestInteriorShading'/>
+              <SummerShadingCoefficient>0.795</SummerShadingCoefficient>
+            </InteriorShading>
+            <Overhangs>
+              <Depth>1.5</Depth>
+              <DistanceToTopOfWindow>10.0</DistanceToTopOfWindow>
+              <DistanceToBottomOfWindow>12.0</DistanceToBottomOfWindow>
+            </Overhangs>
+            <AttachedToWall idref='WallBedroom3West'/>
+          </Window>
+          <Window>
+            <SystemIdentifier id='WindowBathroom2West'/>
+            <Area>5.0</Area>
+            <Orientation>west</Orientation>
+            <UFactor>0.57</UFactor>
+            <SHGC>0.56</SHGC>
+            <ExteriorShading>
+              <SystemIdentifier id='WindowBathroom2WestExteriorShading'/>
+              <SummerShadingCoefficient>0.8</SummerShadingCoefficient>
+            </ExteriorShading>
+            <InteriorShading>
+              <SystemIdentifier id='WindowBathroom2WestInteriorShading'/>
+              <SummerShadingCoefficient>0.795</SummerShadingCoefficient>
+            </InteriorShading>
+            <Overhangs>
+              <Depth>1.5</Depth>
+              <DistanceToTopOfWindow>10.0</DistanceToTopOfWindow>
+              <DistanceToBottomOfWindow>12.0</DistanceToBottomOfWindow>
+            </Overhangs>
+            <AttachedToWall idref='WallBathroom2West'/>
+          </Window>
+          <Window>
+            <SystemIdentifier id='WindowBedroom2West'/>
+            <Area>10.0</Area>
+            <Orientation>west</Orientation>
+            <UFactor>0.57</UFactor>
+            <SHGC>0.56</SHGC>
+            <ExteriorShading>
+              <SystemIdentifier id='WindowBedroom2WestExteriorShading'/>
+              <SummerShadingCoefficient>0.8</SummerShadingCoefficient>
+            </ExteriorShading>
+            <InteriorShading>
+              <SystemIdentifier id='WindowBedroom2WestInteriorShading'/>
+              <SummerShadingCoefficient>0.795</SummerShadingCoefficient>
+            </InteriorShading>
+            <Overhangs>
+              <Depth>1.5</Depth>
+              <DistanceToTopOfWindow>10.0</DistanceToTopOfWindow>
+              <DistanceToBottomOfWindow>12.0</DistanceToBottomOfWindow>
+            </Overhangs>
+            <AttachedToWall idref='WallBedroom2West'/>
+          </Window>
+        </Windows>
+        <Skylights>
+          <Skylight>
+            <SystemIdentifier id='SkylightFamilyRoom'/>
+            <Area>8.0</Area>
+            <Azimuth>0</Azimuth>
+            <UFactor>1.13</UFactor>
+            <SHGC>0.66</SHGC>
+            <AttachedToRoof idref='Roof1'/>
+            <AttachedToFloor idref='CeilingFamilyRoom'/>
+            <extension>
+              <Curb>
+                <Area>2.8</Area>
+                <AssemblyEffectiveRValue>1.96</AssemblyEffectiveRValue>
+              </Curb>
+              <Shaft>
+                <Area>48.0</Area>
```

Comment:
I don't think this is correct? Should be perimeter x height. Perimeter is 2+2+4+4=12 ft, height is 5 ft, so area is 60 ft2.

Reply:
You're right, for some reason I counted it as 4 ft high, I'm changing it.